pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -254,7 +254,7 @@ func (e *EndpointController) deletePod(obj interface{}) {
 	}
 	pod, ok := tombstone.Obj.(*v1.Pod)
 	if !ok {
-		utilruntime.HandleError(fmt.Errorf("Tombstone contained object that is not a Pod: %#v", obj))
+		utilruntime.HandleError(fmt.Errorf("Tombstone contained object that is not a Pod: %#v", tombstone))
```

Comment:
Does the known type change the way the detail print happens?  I actually don't know, so a golang playground example may help.

Also, this is a change that would need to be reflected in all the controllers to be consistent.